iocore/net/I_NetVConnection.h

```diff
@@ -187,6 +188,8 @@ struct NetVCOptions {
   uint32_t packet_tos;
   uint32_t packet_notsent_lowat;
 
+  TLSSNISupport::HintsFromSNI hintsFromSNI;
```

Comment:
Do you still need this? Since you added `hints_from_sni` to `SNISupport`, you should be able to access the member like `vc->hints_from_sni`.

Reply:
ah. Now I see the relationship here. Corrected this.